Makefile

```diff
@@ -24,8 +24,9 @@ GOMAXPROCS ?= 4
 
 # Default os-arch combination to build
 XC_OS ?= darwin freebsd linux netbsd openbsd solaris windows
-XC_ARCH ?= 386 amd64 arm
-XC_EXCLUDE ?= darwin/arm solaris/386 solaris/arm windows/arm netbsd/arm
+XC_ARCH ?= 386 amd64 arm arm64
+# XC_EXCLUDE "arm64" entries excludes both arm and arm64
+XC_EXCLUDE ?= darwin/arm solaris/386 solaris/arm windows/arm netbsd/arm darwin/arm64 freebsd/arm64 netbsd/arm64 openbsd/arm64 solaris/arm64 windows/arm64
```

Comment:
Maybe that comment isn't very clear about this, but what it is trying to explain is that the XC_EXCLUDE list excludes arm entries without having them... that the arm64 entries are enough to exclude both arm and arm64 due to how it matches (it does sub-string matching and "darwin/arm64" matches both "darwin/arm64" and "darwin/arm" as "arm" is a sub-string of "arm64").

If you compare the XC_EXCLUDE list here to the one in consul-template you'll see it has no "arm" entries, just "arm64" entries and the same can be done here.